### PR TITLE
Feat: Updated Alerts

### PR DESCRIPTION
**Pull request by hussiantabha**

### CODE REVIEW

**Comment by Manju-sree** [comment]
Everything looks fine to me with alerts component..

**Review by munjalSaksham71 — COMMENTED** [review]
Looks Good

File: app.css [hunk 1]
```diff
@@ -158,3 +159,56 @@ a{
...
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
+.alert{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 0px solid black;
```
Comment (Priya-Patel31) [inline app.css]: you can use border:none instead of this !!

File: app.css [hunk 2]
```diff
@@ -158,3 +159,56 @@ a{
...
+    width: 350px;
+    margin: 0.75rem 0;
+    border-radius: 0.25rem;
+    flex-wrap: wrap;
+    flex-grow: 1;
+    padding: 1rem 1rem;
+}
+.alert-primary{
+    background-color: #CCE5FF;
+    color: #0064B8;
```
Comment (Priya-Patel31) [inline app.css]: don't use colors directly rather than use var(--color-name) !!

File: app.css [hunk 3]
```diff
@@ -158,3 +159,56 @@ a{
...
+}
+.alert-secondary{
+    background-color: #E2E3E5;
```
Comment (Priya-Patel31) [inline app.css:188]: don't use colors directly rather than use var(--color-name) !!

File: app.css [hunk 4]
```diff
@@ -158,3 +159,56 @@ a{
...
+.alert-danger{
+    background-color: #F8D7DA;
+    color: #721C24;
+}
+.alert-warning{
+    background-color: #FFF3CD;
+    color: #856404;
+}
+.alert-info{
+    background-color: #D1ECF1;
```
Comment (Priya-Patel31) [inline app.css:204]: don't use colors directly rather than use var(--color-name) !!

File: app.css [hunk 5]
```diff
@@ -158,3 +159,56 @@ a{
...
+    color: #0C548F;
+}
+.alert-light{
+    background-color: #FEFEFE;
+    color:#818192;
+}
+.alert-dark{
+    background-color: #D6D8D9;
```
Comment (Priya-Patel31) [inline app.css:212]: don't use colors directly rather than use var(--color-name) !!

**Comment by t007rushi** [comment]
Looks Good made nice color variations, one thing just add light grey border to white BG Alert else merge it

File: app.css [hunk 6]
```diff
@@ -158,3 +159,56 @@ a{
     height: 10rem;
     border-radius: 100%;
 }
+
+/* alerts */
+.alert-div{
```
Comment (Priya-Patel31) [inline app.css]: make different CSS file for every component rather than combining it with app.css

File: app.css [hunk 7]
```diff
@@ -158,3 +159,56 @@ a{
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 0px solid black;
+    width: 350px;
+    margin: 0.75rem 0;
+    border-radius: 0.25rem;
+    flex-wrap: wrap;
+    flex-grow: 1;
+    padding: 1rem 1rem;
```
Comment (Priya-Patel31) [inline app.css]: just a suggestion can use padding : 1rem ;

File: app.css [hunk 8]
```diff
@@ -158,3 +159,56 @@ a{
     height: 10rem;
     border-radius: 100%;
```
Comment (Priya-Patel31) [inline app.css]: there is no need of border-radius property to be used of !!

**Review by Priya-Patel31 — CHANGES_REQUESTED** [review]

File: app.css (hunk 1 above)
Comment (hussiantabha) [reply app.css]: Done Thanks

File: app.css (hunk 6 above)
Comment (hussiantabha) [reply app.css]: Yes done!

File: app.css (hunk 8 above)
Comment (hussiantabha) [reply app.css]: If I don't add this then my picture wouldn't get rounder.

File: app.css (hunk 7 above)
Comment (hussiantabha) [reply app.css]: Done Thanks!

**Pull request merged**